process/best_practices_badge.md

```diff
@@ -0,0 +1,57 @@
+---
+parent: Processes
+has_children: true
+---
+
+# OpenSSF Best Practices Badge
+
+* TOC
+{:toc}
+
+
+The [Open Source Security Foundation (OpenSSF) Best Practices badge](https://bestpractices.coreinfrastructure.org/en) allows Free/Libre and Open Source Software (FLOSS) projects to show that they follow best practices in managing their project. Consumers of the badge can quickly assess which FLOSS projects are following best practices and, as a result, are more likely to produce higher-quality secure software. Academy Software Foundation projects, as a part of the [project lifecycle established by the TAC](lifecycle), achieve this badge to showcase them being exemplars in open source for the visual effects and motion picture industries. 
+
+This document outlines specific questions Academy Software Foundation projects have raised in regard to fulfilling the badge requirements. If you have questions not answered, please provide a PR that adds your question below and will be merged with an answer.
+
+## "The project MUST have at least one primary developer who knows how to design secure software." - how to fulfill this?
+
+Having a primary developer take the three courses outlined below ( all free to audit ) would be sufficient:
```

Comment:
I recently found https://github.com/ossf/wg-best-practices-os-developers/blob/main/docs/Concise-Guide-for-Developing-More-Secure-Software.md and https://github.com/ossf/wg-best-practices-os-developers/blob/main/docs/Existing%20Guidelines%20for%20Developing%20and%20Distributing%20Secure%20Software.md, which have a bunch of interesting links.

Reply:
Great references! Added then in now.